extensions/amp-sidebar/0.1/amp-sidebar.css

```diff
@@ -26,24 +26,33 @@ amp-sidebar {
   overflow-y: auto !important;
   z-index: 9999 !important;
   -webkit-overflow-scrolling: touch;
+  /*
+   * Override layout="nodisplay".
+   * We use translateX and aria-hidden for a11y to hide the sidebar and then
+   * simply translate it back as a separate composition layer.
+   * Note that since sidebar takes owenership of its children, this does
+   * not cause layout for children until sidebar is opened.
+   */
```

Comment:
this is not possible until the sidebar can schedule pause and layout for its children